docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -84,6 +86,19 @@ gradlew eclipse
 . https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
 . Start creating your tests
 
+=== Visual Studio Code
+
+==== Prerequisites
+
+. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studiop[VIVIDUS Studio]
+
+==== Open the project in Visual Studio Code
```

Comment:
what about tests execution?

Reply:
Good idea but I would prefer to put this into extension

Reply:
I think it should be here, despite the absence of this section in eclipse :)